app/Jobs/SyncDiscordMember.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Member;
+use App\Services\AODClient;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Queue\Queueable;
+
+class SyncDiscordMember implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(public Member $member) {}
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        $botAPIResponse = $this->member->forumMember();
+
+        if (! empty($botAPIResponse->discordid)) {
```

Comment:
We probably need an initial check to verify that `$botAPIResponse` actually returned an object (or null) which prevents the rest of this from working

Reply:
Ah you're right. I didn't think response codes would throw an exception, but it looks like that behavior only stops if `http_errors` is set to false on the request. Cool!